### DISCUSSION TITLE
VCW 254/2 and VRT 390

### DISCUSSION THREAD

**Discussion by andmoos** [john30/ebusd#621] [Q&A] [answered]
Hi, 

I do have an older model of a Vailant VCW 254/2 with an VRT 390 room thermostat.

I found  the VRT 390 features a 3-Pin "7 8 9" connector as used on some more recent Vailant ebus devices I found images online of. But there is no distinct mention of "ebus" anywhere for the VRT 390 nor the VCW 254/2.

So my two questions: 
 1. Does anyone know if these devices use ebus at all?
 2. If they do use ebus, can they be used with ebusd + ebus adapter?

Any help appreciated, thanks in advance!

**Comment by john30** [comment, marked as answer]
7-8-9 is not eBUS, so you'd have to have some kind of adapter for triggering heating and getting feedback through this.
if any of the devices has an eBUS connector on the board, then it is usually possible to listen to the traffic and if all goes well also to actively read some data (when there is a config for the device).

**Reply by andmoos** [reply]
Ok, thanks for your time to clarify (and sorry for the late reply).

For anybody coming around this question as well, the most comprehensive information about the 789 interface I found so far was in this German Forum discussion: https://www.mikrocontroller.net/topic/12625
